cognite/client/_api/files.py

```diff
@@ -484,7 +485,13 @@ def upload(
         raise ValueError(f"The path '{path}' does not exist")
 
     def _upload_file_from_path(self, file: FileMetadata, file_path: str, overwrite: bool) -> FileMetadata:
+        from cognite.client import _RUNNING_IN_BROWSER
```

Comment:
I'd suggest placing this constant somewhere else than `cognite/client/__init__.py` so you don't need to use local imports every time you need to use it.

Reply:
Yes, very yes 😅